src/sysen3/papsav.1

```diff
@@ -0,0 +1,372 @@
+	title papsav
+
+	; source reconstructed by EJS on 2018-12-15 from CHANNA;RAKASH PAPSAV binary
+	
+t1=1
+t=2
+tt=3
+a=4
+.prsva=4
+b=5
+c=6
+f=7
+pt=10
+ch=11
+count=12
+.prsvz=13
+.ap.=14
+.pf.=15
+.cf.=16
+p=17
+
+TYIC==2 		; TTY INPUT CHANNEL
+TYOC==1 		; tty output channel
+TYO=<.IOT TYOC,> 	; INSTRUCTION TO OUTPUT A CHAR
+;;; MAIN PROGRAM
+
+i==4
+l==5
+v==6
+lim==7
+remapt==2000
+
+    loc 100
+.entr.: movei tt,t1(p)
+        hrli tt,.prsva
+        add p,.retn.+7
+        blt tt,(p)
+        move .ap.,t1
+        move .pf.,.cf.
+        move .cf.,p
+        add p,(t)
+        jumpl p,t1(t)
+        .suset .retn.+11
+        jrst t1(t)
+.retn.: move p,.cf.
+        move .cf.,.pf.
+        movsi tt,-11(p)
+        hrri tt,.prsva
+        blt tt,.pf.
+        sub p,.retn.+7
+        popj p,
+        count,,count
+        200000
+        setz .retn.+10(.ap.)
+        0
+fmtbl:  0
+        typoct ;1 full word octal
+        typdec ;2 full wd decimal
+        typhaf ;3 " with commas
+        tycrlf ;4 do cr
+        cpopj  ;5 ?
+        typsix ;6 sixbit
+        typasz ;7 asciz
+
+cpopj:  popj p,
+
+feep:   blt b,(.prsva)
+crlf:   asciz/
+/
+
+CMACMA:	ASCIZ/,,/
+
+SEP:	ASCIZ/-----
+/
+
+bletch: asciz\
+WRONG SYSTEM VERSION - REPURIFY
+\
+
+MEMHOL:	ASCIZ\
+WARNING: THERE IS A HOLE IN MEMORY
+\
+
+	loc 164
+
+STACK=100		;DEFAULT TO 100 WORDS
+pdl: 	BLOCK STACK
+
+	loc 264		; not sure why removing this causes errors
+
+sysmsg: move .cf.,[-stack,,memhol+7]
+	move p,[-stack,,memhol+7]
+	.call [setz ? sixbit/OPEN/ ? [1,,1] 
```

Comment:
[.ua0,,tyoc]

Reply:
are you saying to change the [1,,1] to [.uao,,tyoc]?

Reply:
Yes.  The first argument to OPEN specifies a mode and a channel.